packages/block-editor/src/components/panel-color-settings/README.md

```diff
@@ -0,0 +1,42 @@
+# `PanelColorSettings`
+
+Render a panel with color choices shown in circles, allowing a user to pick them.
+
+## Properties
+
+### `children`
+
+* **Type:** `Element`
+* *Required:*  No
+* *Default:* Null
+
+The content to be displayed within the panel.
```

Comment:
Specifically below the color palettes. It could also be worth mentioning that this is useful for rendering a `ContrastChecker` within the panel.